pkg/command/command.go

```diff
@@ -0,0 +1,38 @@
+package command
+
+import (
+	"fmt"
+	"os/exec"
+)
+
+type Commander interface {
+	Execute(string, ...string) ([]byte, error)
+	ExecuteWithEnv(string, map[string]string, ...string) ([]byte, error)
+}
+
+type Command struct {
+}
+
+func NewCommand() Commander {
+	return &Command{}
+}
+
+func (c *Command) Execute(cmd string, args ...string) ([]byte, error) {
+	return execute(cmd, args, map[string]string{})
+}
+
+func (c *Command) ExecuteWithEnv(cmd string, envs map[string]string, args ...string) ([]byte, error) {
```

Comment:
Might not be necessary right now, but there is a very nice options pattern adopted in go. Might be useful for this case:
https://dev.to/kittipat1413/understanding-the-options-pattern-in-go-390c